Objective-C.gitignore

```diff
@@ -34,6 +34,7 @@ xcuserdata
 # http://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
 #Pods/
+#Podfile.lock
```

Comment:
From [this link](http://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control) I can see the official advice is:

> Whether or not you check in the Pods directory, the **Podfile** and **Podfile.lock** should always be kept under version control.